### PR TITLE
feat: inventory webhook

### PR DESCRIPTION
**Pull request by merlin-northern**
Changelog: Title
Ticket: MEN-7451

### CODE REVIEW

File: api/http/management.go [hunk 1]
```diff
@@ -88,6 +88,11 @@ func (h *ManagementHandler) CreateIntegration(c *gin.Context) {
 		)
 		return
 	}
+	// we introduced a scope of integration and all the previous
+	// integrations are in the `deviceauth` scope
+	if len(integration.Scope) < 1 {
+		integration.Scope = model.ScopeDeviceAuth
+	}
```
Comment (merlin-northern) [inline api/http/management.go:95]: in here I will enforce the professional plan (somehow), so only those tenants can create inventory webhooks.

**Comment by merlin-northern** [comment]
@alfrunes @kjaskiewiczz please give me early review on the architecture. the code needs some cleaning and there are some loose ends to tie. goes with https://github.com/mendersoftware/inventory-enterprise/pull/551

**Comment by kjaskiewiczz** [comment]
I guess this one can be closed, right?

**Comment by merlin-northern** [comment]
> I guess this one can be closed, right?

right.

**Pull request closed without merging**